docs/Bitbucket.md

```diff
@@ -0,0 +1,169 @@
+| `definitions.caches.<name>` | ✅ | A customised cache name applicable for one or more steps within a Bitbucket pipeline. |
+| `definitions.caches.<name>.path` | ✅ | The directory path that is desired to be cached. |
+| `definitions.caches.<name>.key.files` | ⚠️ | The list (one or more) files that are monitored for changes - and stored once its hash changes between file versions change. If multiple files are specified - multiple cache-plugin definitions are set on the resultant Buildkite command step (differing `manifest` properties between each). |
+| `definitions.pipeline` | ⚠️ | Pipelines that are exported for re-use within repositories of the same workspace. Like functionality exists within Buildkite as [Pipeline Templates](https://buildkite.com/docs/pipelines/templates). |
+| `definitions.services` | ✅ | Defined Docker services that are applied within a Bitbucket pipeline. Services defined in a corresponding Bitbucket pipeline step using the `pipelines.default.step.services` property will have its configuration applied with the use of the [docker-compose-buildkite-plugin](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin). <br/><br/> Generated configuration will need to be saved to a `compose.yaml` file within the repository, and the image utilised with the Buildkite command step as `app`. <br/><br/> Refer to the Bitbucket pipelines [documentation](https://support.atlassian.com/bitbucket-cloud/docs/databases-and-service-containers/) for more details on service containers and configuration references. |
+
+## Export (`export`)
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `export` | ❌ | Bitbucket Premium option of sharing pipeline configuration between workspaces. Not applicable within Buildkite as an attribute - like functionality exists within [Pipeline Templates](https://buildkite.com/docs/pipelines/templates). |
+
+## Image (`image`)
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `image` | ✅ | The container image that is to be applied for each step within a Bitbucket pipeline, utilising the specified image within the [docker-buildkite-plugin](https://github.com/buildkite-plugins/docker-buildkite-plugin). This has lower precedence over per-step `image` configuration (see `pipelines.default.step.image`). <br/><br/> The `aws`, `aws.oidc`, `name`, `username` and `password` sub-properties are supported - refer to the Bitbucket pipelines [documentation](https://support.atlassian.com/bitbucket-cloud/docs/docker-image-options/) for more information. |
```

Comment:
I would change the comment to the actual translation not the original configuration.

```suggestion
| `image` | ✅ | The container image that is to be applied for each step within a Bitbucket pipeline, utilising the specified image within the [docker-buildkite-plugin](https://github.com/buildkite-plugins/docker-buildkite-plugin). This has lower precedence over per-step `image` configuration (see `pipelines.default.step.image`). <br/><br/> The `aws`, `aws.oidc`, `name`, `username` and `password` sub-properties are supported through the use of the corresponding plugin ([Docker Login](https://github.com/buildkite-plugins/docker-login-buildkite-plugin/) or [ECR](https://github.com/buildkite-plugins/ecr-buildkite-plugin/)). |
```

Reply:
Altered - with the same liner referenced in the `steps` property also